sqlconnect/op/operator.go

```diff
@@ -0,0 +1,47 @@
+package op
+
+type Operator string
+
+const (
+	// =
+	Eq Operator = "eq"
+	// !=
+	Neq Operator = "neq"
+	// IN (...)
+	In Operator = "in"
+	// NOT IN (...)
+	NotIn Operator = "notin"
+
+	// >
+	Gt Operator = "gt"
+	// >=
+	Gte Operator = "gte"
+	// <
+	Lt Operator = "lt"
+	// <=
+	Lte Operator = "lte"
+
+	// LIKE
+	Like Operator = "like"
+	// NOT LIKE
+	NotLike Operator = "notlike"
+	// left <= v <= right
+	Between Operator = "between"
+	// v < left OR v > right
+	NotBetween Operator = "notbetween"
+	// left >= now() - INTERVAL right
+	NbfInterval Operator = "nbfinterval"
```

Comment:
The key nbfinterval is not clear to me. Can we give it a clearer name?

Reply:
Open to suggestions! `nbf` prefix stands for not before.

Reply:
Instead can we have something like
beforeintervalbounded & beforeintervalunbounded
afterintervalbounded & afterintervalunbounded

Reply:
Also, we can use camelCase or snake case operators. Will make things much clearer. Especially in case we are using bigger ones.

Reply:
imo operators should be kept small and case insensitive

Reply:
what about:
gti --> gt an interval
gtei --> gte an interval
lti --> less than an interval
ltei --> less than an interval
betweeni --> between an interval

Reply:
Another option would be to keep using gt, gte, lt, lte, between, etc. and allow a special value format for intervals, e.g. `"now-1d"`

Reply:
finally we can keep ti simple... will name it `inlast`